token-lending/program/src/instruction.rs

```diff
@@ -283,6 +285,29 @@ pub enum LendingInstruction {
         /// The new owner
         new_owner: Pubkey,
     },
+    // 12
+    /// Make a flash loan.
+    ///
+    ///   0. `[writable]` Destination liquidity token account, minted by reserve liquidity mint.
+    ///   1. `[writable]` Reserve account.
+    ///   2. `[]` Lending market account.
+    ///   3. `[]` Derived lending market authority.
+    ///   4. `[]` Temporary memory
```

Comment:
This doesn't look like it's used anywhere.

Reply:
Thanks! Removed